pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -163,12 +163,6 @@ func (m *Manager) AddResourceGroup(grouppb *rmpb.ResourceGroup) error {
 	if grouppb.GetPriority() > 16 {
 		return errs.ErrInvalidGroup
 	}
-	m.RLock()
-	_, ok := m.groups[grouppb.Name]
-	m.RUnlock()
-	if ok {
-		return errs.ErrResourceGroupAlreadyExists.FastGenByArgs(grouppb.Name)
```

Comment:
Need to remove the `ErrResourceGroupAlreadyExists` related error definition.

Reply:
done.